sub.go

```diff
@@ -72,6 +73,11 @@ func (sub *subSocket) Type() SocketType {
 	return sub.sck.Type()
 }
 
+// Addr returns the listen address
```

Comment:
please adapt with whatever updated `zmq4.Socket.Addr` documentation you came up with.